docker-compose.yml

```diff
@@ -55,6 +48,30 @@ services:
     environment:
       - POSTGRES_USER=adapterservice
       - POSTGRES_PASSWORD=admin
+    command:
+      - '-cwal_level=logical'
+      - '-cmax_wal_senders=1'
+      - '-cmax_replication_slots=1'
```

Comment:
What's that about?

Reply:
Debezium uses [Postgres's logical decoding feature](https://www.postgresql.org/docs/current/logicaldecoding-explanation.html) to get the write-ahead log content. This enables and configures that feature.

Reply:
please add a comment for that! ;-)

Reply:
Done!